runtimes/huggingface/tests/test_tasks/test_japanese_mask_fill.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from mlserver.settings import ModelSettings, ModelParameters
+from mlserver_huggingface import HuggingFaceRuntime
+from mlserver_huggingface.codecs import HuggingfaceRequestCodec
+
+
+test_sentence = "実際に空が[MASK]のか？"
+
+
+@pytest.fixture
+def current_model_settings(request) -> ModelSettings:
```

Comment:
nit: rename fixture to be a bit more descriptive of the fact that the tokenizer is different?